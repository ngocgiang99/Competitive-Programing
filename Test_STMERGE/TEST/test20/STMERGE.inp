6
9 8
896589045 30571654 65247401 133431335 410994449 157618007 59058069 107444272 
618198690 32455442 761658920 853326111 703259164 735367358 785123062 861729305 
864913971 721594271 37151956 360726146 736969182 688551737 950129368 602338304 
384854096 400490836 355476054 548613751 766219007 728567282 399293668 77655053 
987023264 168930649 855545941 589841191 635746344 230174734 347119297 566314602 
204644044 388324567 943752826 288913594 819701249 592018616 866278935 693659452 
172033147 90979574 467455085 906616151 965874481 184211734 592251024 404047022 
267692886 567877341 147741653 692049190 754906209 931332688 827818450 841222373 
653794821 604420021 846227547 476583504 205506104 560749455 565780357 454347276 
7 3
58161062 512664829 896206175 
515443195 529829202 934066761 
404189564 799706838 493058399 
66171732 658987645 27553467 
301463265 844606404 998085526 
349119343 478311213 915320190 
910163515 971058424 442123225 
7 2
421561241 890748089 
139293091 211190081 
244049369 262684130 
934667543 515209675 
440185468 404170154 
831102441 440594486 
579218567 618351270 
6 2
612951871 455871401 
557116539 569798594 
197381161 973812170 
13367108 593108238 
445803787 660102238 
382013823 505086094 
8 5
663359825 849312761 767673607 246364016 165861662 
108006722 897668332 642343897 535387019 62639263 
922154588 17414792 183208319 76985990 15448156 
824350226 405586973 184055210 253333041 444108109 
128391371 2917632 458441568 354447693 234965293 
496992703 781403055 321998184 226725161 100106587 
400143138 485988146 9497316 573490056 835650045 
861468288 52820162 414129213 83208391 246499688 
9 7
154930245 527207486 563494333 526731343 6015354 906607000 804984549 
586323670 892548012 830993482 284058691 838188600 597338209 766505833 
715050879 312729002 611673436 794964555 718660220 108141216 873924772 
888995460 498696544 754655953 699166333 473613801 889552015 956854833 
800570732 994784912 122777263 417571060 370386486 615142093 9128101 
157698961 379652065 378069302 430679957 135119547 815410608 34669215 
625510117 636109672 466026794 509240350 661927994 632757965 627983379 
318177030 469494648 875994207 384729200 909883631 104503493 409952211 
824952966 529701803 966030981 900808989 902140122 229174609 172806494 
