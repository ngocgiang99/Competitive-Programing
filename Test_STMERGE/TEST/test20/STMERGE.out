107444272
431743031
401977221
382013823
123454878
235189963
